support/Environments/caltech_hpc_gcc_skylake.sh

```
@@ -13,12 +13,12 @@ spectre_setup_modules() {
 
 spectre_load_modules() {
     module use /central/groups/sxs/modules/
-    module load libraries/spectre-deps/skylake-2024-02
+    module load libraries/spectre-deps/skylake-2024-03
```

Comment about the _Icelake_ environment: there's a `> /dev/null 2>&1` that silences any error messages. This also silences any helpful output when I have conflicting modules already loaded and try to run `spectre_load_modules`. Maybe just remove?

---

Ok removed. I originally had that because the intel MPI module kept printing stuff whenever it was loaded/unloaded which I found annoying, but I found a way to silence it in the intel MPI module file